wiki/en/Running-a-Server.md

```diff
@@ -83,48 +83,150 @@ If you want to run a number of Servers, possibly also behind a firewall or on a
 
 To run a Custom Directory [read this guide](Custom-Directories)
 
-# Bandwidth use
 
-## Audio bandwidth
+# Installing and running a Server 
 
-The audio settings have an impact on the required network bandwidth. The table below summarises network requirements with respect to the configuration of:
-* Channels : stereo/mono
-* Quality : high/medium/low
-* Audio buffer duration : 2.67 ms, 5.33 ms, 10.67 ms, 21.33 ms
+Most people run Jamulus as a "pure" Server on **hardware without audio** (e.g. on a 3rd party/cloud host) running Linux. The following guide assumes you are familiar with the command line and Debian/Ubuntu or similar distribution using systemd. To run a server on Windows or on the desktop with a graphical user interface, [see this section](#servers-on-the-desktop).  
```

Comment:
To run a server on Windows/macOS on the desktop with a graphical user interface

Reply:
> To run a server on Windows/macOS on the desktop with a graphical user interface

Are you intending to _exclude_ Linux desktop?

Reply:
No. Not really.

Reply:
This is why I purposefully _didn't_ mention the platform :-)